File: scratch/check_config.py
import os
from dotenv import load_dotenv
from supabase import create_client

load_dotenv('.env')
url = os.environ.get("NEXT_PUBLIC_SUPABASE_URL")
key = os.environ.get("NEXT_PUBLIC_SUPABASE_ANON_KEY")

supabase = create_client(url, key)
res = supabase.table('league_config').select('*').single().execute()
print("Config:", res.data)
res2 = supabase.table('fixtures').select('id, matchday, momento, start_time').order('start_time').limit(5).execute()
print("Fixtures:", res2.data)

File: scratch/check_config.js
import { createClient } from '@supabase/supabase-js';
import fs from 'fs';
const env = fs.readFileSync('.env', 'utf-8').split('\n');
let url = '', key = '';
env.forEach(line => {
  if (line.startsWith('NEXT_PUBLIC_SUPABASE_URL=')) url = line.split('=')[1].trim();
  if (line.startsWith('NEXT_PUBLIC_SUPABASE_ANON_KEY=')) key = line.split('=')[1].trim();
});
const supabase = createClient(url, key);
async function run() {
  const { data: cfg } = await supabase.from('league_config').select('*').single();
  console.log('league_config:', cfg);
  const { data: fixtures } = await supabase.from('fixtures').select('*').order('start_time').limit(5);
  console.log('First 5 fixtures:', fixtures.map(f => ({id: f.id, matchday: f.matchday, start: f.start_time})));
}
run();
